### PR TITLE
New Note script

### PR DESCRIPTION
**Pull request by commai**
## Description

Simply allows you to create a new note with appropriate title and body formatting via Raycast

## Type of change
- [x] New script command

## Screenshot

https://user-images.githubusercontent.com/46573539/126587604-fc52ea20-d7b6-4c00-8a04-e748937b0868.mov

## Dependencies / Requirements

Simply add it to your list of enabled commands 

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/apps/notes/QuickNote.applescript [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# @raycast.title Quick Notes
+# @raycast.mode silent
+
+# Optional parameters:
+# @raycast.icon ./images/notes.png
+# @raycast.argument1 { "type": "text", "placeholder": "Note Name" }
+# @raycast.argument2 { "type": "text", "placeholder": "Text" }
+
+# Documentation:
+# @raycast.description Create a New Note quickly via Raycast
```
Comment (unnamedd) [inline commands/apps/notes/QuickNote.applescript:14]: As the Script Commands are done to work primarily on Raycast, doesn't make much sense mention "via Raycast" in the title of the Script Command.

```suggestion
# @raycast.description Create a new Note quickly 
```

File: commands/apps/notes/QuickNote.applescript [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/osascript
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Quick Notes
```
Comment (unnamedd) [inline commands/apps/notes/QuickNote.applescript:5]:
```suggestion
# @raycast.title Create Note
```

File: commands/apps/notes/QuickNote.applescript [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+# @raycast.mode silent
+
+# Optional parameters:
+# @raycast.icon ./images/notes.png
+# @raycast.argument1 { "type": "text", "placeholder": "Note Name" }
+# @raycast.argument2 { "type": "text", "placeholder": "Text" }
+
```
Comment (unnamedd) [inline commands/apps/notes/QuickNote.applescript:12]: One of our requirements for the submission of Script Commands to this repository is the use of the meta tag `packageName` to improve portability. You can check some of our conventions [here](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md#metadata-convention).

```suggestion
# @raycast.package Notes

```

File: commands/apps/notes/QuickNote.applescript [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+# Documentation:
+# @raycast.description Create a New Note quickly via Raycast
+# @raycast.author Vardan Sawhney
+# @raycast.authorURL https://github.com/commai
+
+on run argv
+	set content to "<body><h1 style=\"color:black;\"> " & (item 1 of argv)  & "</h1> 
+	<p style=\"color:black;\" >" & (item 2 of argv) & "</p>
+	</body>"
```
Comment (unnamedd) [inline commands/apps/notes/QuickNote.applescript:21]: Really interesting! I didn't know under the hood Notes used HTML! Thanks for the TIL!

**Review by unnamedd — CHANGES_REQUESTED** [review]
Thanks for the Script Command @commai! Please, check my comments and let me know when finish to address them! 🤘

**Comment by commai** [comment]
@unnamedd Just finished making the fixes! Thanks for reviewing :)

**Review by grzegorzkrukowski — CHANGES_REQUESTED** [review]
Thanks for your contribution.

It all looks good, but one small thing is to follow up [File naming convention](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md#file-naming-convention) - please use dashed, lower-case naming for the file.

**Comment by unnamedd** [comment]
> Thanks for your contribution.
> 
> It all looks good, but one small thing is to follow up [File naming convention](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md#file-naming-convention) - please use dashed, lower-case naming for the file.

Excellent point @grzegorzkrukowski!

File: commands/apps/notes/create-note.applescript [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Create Note
+# @raycast.mode silent
+
+# Optional parameters:
+# @raycast.icon ./images/notes.png
+# @raycast.argument1 { "type": "text", "placeholder": "Note Name" }
+# @raycast.argument2 { "type": "text", "placeholder": "Text" }
+# @raycast.package Notes
```
Comment (unnamedd) [inline commands/apps/notes/create-note.applescript:12]:
```suggestion
# @raycast.packageName Notes
```

**Review by unnamedd — CHANGES_REQUESTED** [review]
@commai just missing a last point and then we will be good to go!

**Review by RSO — APPROVED** [review]

**Comment by RSO** [comment]
Thank you for your contribution!

**Pull request merged**